test/withStyles_test.jsx

```diff
@@ -305,4 +310,48 @@ describe('RTL support', () => {
       expect(resolveNoRTLStub.callCount).to.equal(1);
     });
   });
+
+  describe('contextual create', () => {
+    it('calls ThemedStyleSheet.createLTR without direction set', () => {
+      function MyComponent() {
+        return null;
+      }
+
+      const WrappedComponent = withStyles(() => ({}))(MyComponent);
+      render(
+        <DirectionProvider direction={DIRECTIONS.LTR}>
+          <WrappedComponent />
+        </DirectionProvider>,
+      );
+      expect(testInterface.create.callCount).to.equal(1);
+    });
+
+    it('calls ThemedStyleSheet.createLTR with LTR direction', () => {
+      function MyComponent() {
+        return null;
+      }
+
+      const WrappedComponent = withStyles(() => ({}))(MyComponent);
+      render(
+        <DirectionProvider direction={DIRECTIONS.LTR}>
+          <WrappedComponent />
+        </DirectionProvider>,
+      );
+      expect(testInterface.create.callCount).to.equal(1);
+    });
+
+    it('calls ThemedStyleSheet.createRTL with RTL direction', () => {
+      function MyComponent() {
+        return null;
+      }
+
+      const WrappedComponent = withStyles(() => ({}))(MyComponent);
+      render(
+        <DirectionProvider direction={DIRECTIONS.RTL}>
+          <WrappedComponent />
+        </DirectionProvider>,
+      );
+      expect(testInterface.createRTL.callCount).to.equal(1);
+    });
+  });
```

Comment:
nit: please add one more test for a `<WrappedComponent />` that is not within a `DirectionProvider`?

Reply:
Whoops that's what I meant the first test to be :P